src/app.py

```diff
@@ -36,14 +44,11 @@ def sleep_data(garth_client: Client) -> dict:
     Returns:
         dict: The daily sleep data for the user.
     """
-    try:
-        return garth_client.connectapi(
-            f"/wellness-service/wellness/dailySleepData/{garth_client.username}",
-            params={"date": datetime.date.today().isoformat(),
-                    "nonSleepBufferMinutes": 60},
-        )
-    except Exception as e:
-        print(e)
```

Comment:
Here you're handling all possible exceptions, printing the entire exception and returning `None`.